docs/advanced-topics/message-system/clientrpc.md

```diff
@@ -49,7 +49,7 @@ void Pong(int somenumber, string sometext) { /* ... */ }
 void PongClientRpc(int somenumber, string sometext) { /* ... */ }
 ```
 
-The `[ClientRpc]` attribute and matching `...ClientRpc` suffix in the method name are there to ensure RPC call sites know when they're executing an RPC. The client RPC replicates and executes client-side, without jumping into the original RPC method declaration to find out if it's an RPC.
+The `[ClientRpc]` attribute and matching `...ClientRpc` suffix in the method name ensure RPC call sites know when they're executing an RPC. The client RPC replicates and executes client-side, without jumping into the original RPC method declaration to find out if it's an RPC.
```

Comment:
> The `[ClientRpc]` attribute and matching `...ClientRpc` suffix in the method name help to assure both context clarity in scripts that invoke them and are used by Netcode during the ILPostProcessor pass.  The ILPostProcessor pass replaces all call-site locations, where the RPC method is invoked, with additional code generated specific to the RPC to assure that the RPC message is generated and sent to the appropriate destinations/targets.

Something like the above I think might work?

Reply:
Love it!